### PR TITLE
Add voting wallet UI

### PR DESCRIPTION
**Pull request by coreyar**
Adds UI for Voting Wallet section on the vote page

### CODE REVIEW

File: src/pages/Vote/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import Vote from '.';
+
+export default {
+  title: 'Pages/Vote',
+  component: Vote,
+} as ComponentMeta<typeof Vote>;
+
+export const Default = () => <Vote />;
```
Comment (therealemjy) [inline src/pages/Vote/index.stories.tsx:12]: We'll need to wrap this story with a `Router` tag:
Screen Shot 2022-05-31 at 9 00 14 AM

File: src/theme/MuiThemeProvider/muiTheme.ts [hunk 2]
```diff
@@ -121,6 +121,7 @@ export default createTheme({
     h4: {
       fontSize: '1.25rem',
       fontWeight: 600,
+      lineHeight: '1.875rem',
```
Comment (therealemjy) [inline src/theme/MuiThemeProvider/muiTheme.ts:127]: This should be the default line-height of all text elements (1.5)

**Comment by therealemjy** [comment]
I can't check the page in Storybooks for now because it needs to be wrapped with a `Router` tag. Once fixed I can have another look!

File: src/theme/MuiThemeProvider/muiTheme.ts (hunk 2 above)
Comment (coreyar) [reply src/theme/MuiThemeProvider/muiTheme.ts:127]: According to the designs h4 has a larger lineHeight of 30px

Comment (therealemjy) [reply src/theme/MuiThemeProvider/muiTheme.ts:127]: It's not larger: 30 is 20 * 1.5, 1.5 being the line height we apply to all elements

File: src/pages/Vote/VotingWallet/index.tsx [hunk 3]
```diff
@@ -1,20 +1,82 @@
...
+}
 
+export const VoteUi: React.FC<IVoteUiProps> = ({ votingWeight, xvsLockedWei }) => {
+  const { t, Trans } = useTranslation();
+  const styles = useStyles();
+  const readableXvsLocked = convertWeiToCoins({
+    valueWei: xvsLockedWei,
+    tokenId: 'xvs',
+    returnInReadableFormat: true,
+    addSymbol: false,
```
Comment (therealemjy) [inline src/pages/Vote/VotingWallet/index.tsx:24]: We'll also need to minimize decimals:
```suggestion
    addSymbol: false,
    minimizeDecimals: true,
```

File: src/components/v2/Button/index.tsx [hunk 4]
```diff
@@ -50,3 +56,12 @@ export const PrimaryButton = (props: IButtonProps) => <Button variant="primary"
...
 export const TextButton = (props: IButtonProps) => <Button variant="text" {...props} />;
+export const LinkButton = ({
+  variant = 'primary',
+  fullWidth = false,
+  small = false,
+  ...props
+}: LinkProps & IBaseButtonProps) => {
+  const styles = useStyles({ fullWidth, variant, small });
+  return <Link {...props} css={styles.getButton({ disabled: false })} />;
+};
```
Comment (therealemjy) [inline src/components/v2/Button/index.tsx:67]: We should add this variant to the story

**Review by therealemjy — COMMENTED** [review]
Spotted as few design issues:
- text inside `LinkButton` has the wrong font-weight (should be the same for other buttons)
- "Voting weight" block has a top padding that isn't present in the designs
- both the "Voting weight" and "Total locked" block have horizontal paddings that aren't present in the designs
- the padding between the "Total locked" label and the divider is 24px but should be 16px
- the padding between the "Total locked" block and the submit button is 16px but should be 24px

**Review by therealemjy — APPROVED** [review]

**Pull request merged**